docs/frontend-system/building-apps/01-index.md

```diff
@@ -0,0 +1,124 @@
+---
+id: index
+title: Building Frontend Apps
+sidebar_label: Overview
+# prettier-ignore
+description: Building frontend apps using the new frontend system
+---
+
+> **NOTE: The new frontend system is in alpha and is only supported by a small number of plugins.**
+
+To get set up quickly with your own Backstage project you can create a Backstage App.
+
+A Backstage App is a monorepo setup that includes everything you need to run Backstage in your own environment.
+
+## Creating a new app
```

Comment:
Hmm, the app package is more of a by-product of the `create-app` flow, I'm feeling it's best to keep this bit in getting started docs? Perhaps we can just link to the getting started docs instead and highlight that once the project has been created you'll find the app in `packages/app`?

Reply:
Text updated